include/z64actor.h

```diff
@@ -38,11 +37,45 @@ typedef struct {
     /* 0x1C */ ActorFunc draw; // Draw function
 } ActorInit; // size = 0x20
 
-typedef enum {
-    /* 0 */ ALLOCTYPE_NORMAL,
-    /* 1 */ ALLOCTYPE_ABSOLUTE,
-    /* 2 */ ALLOCTYPE_PERMANENT
-} AllocType;
+/**
+ * @see ACTOR_ALLOC_ABSOLUTE
+ */
+#define ACTOR_OVERLAY_ABSOLUTE_SPACE_SIZE 0x27A0
+
+/**
+ * The actor overlay should be allocated memory for when loading,
+ * and the memory deallocated when there is no more actor using the overlay.
+ *
+ * `ACTOR_ALLOC_` defines indicate how an actor overlay should be loaded.
+ *
+ * @note Bitwise or-ing `ACTOR_ALLOC_` types is not meaningful.
+ * The `ACTOR_ALLOC_` types are 0, 1, 2 but checked against with a bitwise and.
```

Comment:
This sounds more like an enum at this point.

Reply:
Well with an enum I would expect slapping another value (3) after it would work, but currently it would need to be `1 << 2`  (4), and not 3 which has the 1 and 2 bits set...

Reply:
Mm, rather frustrating.

Reply:
if we do it's not due to these enums being actually bitmasks I hope, I'll change 'em :sunglasses: